jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/popover/PopoverBottomTests.java

```diff
@@ -14,7 +14,7 @@
 
 /**
  * Created by Dmitrii Pavlov on 02.10.2019
- * Email: delnote@gmail.com; Skype: Dmitrii Pavlov
+ * Email: delnote@gmail.com;  Skype: Dmitrii Pavlov
```

Comment:
remove please space (or it's better remove that commit, for example by using interactive rebase)